<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="28214" started-at="2017-01-24T14:25:17Z" finished-at="2017-01-24T14:25:45Z">
    <groups>
    </groups>
    <test name="Chrome-es_US-Login Logout" duration-ms="28214" started-at="2017-01-24T14:25:17Z" finished-at="2017-01-24T14:25:45Z">
      <class name="executionEngine.LoginLogout">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:executionEngine.LoginLogout@527308ce]" name="setUp" is-config="true" duration-ms="2776" started-at="2017-01-24T19:55:17Z" finished-at="2017-01-24T19:55:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Login(java.lang.String)[pri:0, instance:executionEngine.LoginLogout@527308ce]" name="Login" duration-ms="25263" started-at="2017-01-24T19:55:20Z" finished-at="2017-01-24T19:55:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: {"method":"xpath","selector":"//*[@id='page-wrapper']/div[2]/div/div/div/div[2]/div[1]/input"}
Command duration or timeout: 12 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'OEIS-D32', ip: '192.168.169.49', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_60'
*** Element info: {Using=xpath, value=//*[@id='page-wrapper']/div[2]/div/div/div/div[2]/div[1]/input}
Session ID: fe20c604-21a5-4666-a2a2-ea6efb778f47
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=35.0.1}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//*[@id='page-wrapper']/div[2]/div/div/div/div[2]/div[1]/input"}
Command duration or timeout: 12 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'OEIS-D32', ip: '192.168.169.49', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_60'
*** Element info: {Using=xpath, value=//*[@id='page-wrapper']/div[2]/div/div/div/div[2]/div[1]/input}
Session ID: fe20c604-21a5-4666-a2a2-ea6efb778f47
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=35.0.1}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at executionEngine.LoginLogout.Login(LoginLogout.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//*[@id='page-wrapper']/div[2]/div/div/div/div[2]/div[1]/input"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'OEIS-D32', ip: '192.168.169.49', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_60'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/ONKAR~1.NIM/AppData/Local/Temp/anonymous2455922041742289100webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///C:/Users/ONKAR~1.NIM/AppData/Local/Temp/anonymous2455922041742289100webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10779)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/ONKAR~1.NIM/AppData/Local/Temp/anonymous2455922041742289100webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/ONKAR~1.NIM/AppData/Local/Temp/anonymous2455922041742289100webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12666)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/ONKAR~1.NIM/AppData/Local/Temp/anonymous2455922041742289100webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12608)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="SKIP" signature="gotoCatalogPage(java.lang.String)[pri:1, instance:executionEngine.LoginLogout@527308ce]" name="gotoCatalogPage" duration-ms="0" started-at="2017-01-24T19:55:45Z" depends-on-methods="executionEngine.LoginLogout.Login" finished-at="2017-01-24T19:55:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginLogout.gotoCatalogPage(java.lang.String)[pri:1, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginLogout.gotoCatalogPage(java.lang.String)[pri:1, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoCatalogPage -->
        <test-method status="SKIP" signature="uploadProductFile(java.lang.String)[pri:2, instance:executionEngine.LoginLogout@527308ce]" name="uploadProductFile" duration-ms="0" started-at="2017-01-24T19:55:45Z" depends-on-methods="executionEngine.LoginLogout.gotoCatalogPage" finished-at="2017-01-24T19:55:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginLogout.uploadProductFile(java.lang.String)[pri:2, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginLogout.uploadProductFile(java.lang.String)[pri:2, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- uploadProductFile -->
        <test-method status="SKIP" signature="uploadSkuFile(java.lang.String)[pri:3, instance:executionEngine.LoginLogout@527308ce]" name="uploadSkuFile" duration-ms="0" started-at="2017-01-24T19:55:45Z" depends-on-methods="executionEngine.LoginLogout.uploadProductFile" finished-at="2017-01-24T19:55:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginLogout.uploadSkuFile(java.lang.String)[pri:3, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginLogout.uploadSkuFile(java.lang.String)[pri:3, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- uploadSkuFile -->
        <test-method status="SKIP" signature="deleteSkuFromProduct(java.lang.String)[pri:5, instance:executionEngine.LoginLogout@527308ce]" name="deleteSkuFromProduct" duration-ms="0" started-at="2017-01-24T19:55:45Z" depends-on-methods="executionEngine.LoginLogout.uploadSkuFile" finished-at="2017-01-24T19:55:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginLogout.deleteSkuFromProduct(java.lang.String)[pri:5, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginLogout.deleteSkuFromProduct(java.lang.String)[pri:5, instance:executionEngine.LoginLogout@527308ce] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteSkuFromProduct -->
        <test-method status="PASS" signature="tearDown(java.lang.String)[pri:0, instance:executionEngine.LoginLogout@527308ce]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-01-24T19:55:45Z" finished-at="2017-01-24T19:55:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- executionEngine.LoginLogout -->
    </test> <!-- Chrome-es_US-Login Logout -->
  </suite> <!-- Suite -->
</testng-results>
